File: RetailStation.Entities/DTOs/Operation/SliderModel.cs
using Microsoft.AspNetCore.Http;
using RetailStation.Entities.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RetailStation.Entities.DTOs.Operation
{
    public class SliderModel : CreatorModel
    {
        public int? SliderId { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public IFormFile Image { get; set; }

        public string Link { get; set; }
        public string ImageURL { get; set; }
        public bool IsActive { get; set; } = true;
        public int? TotalCount { get; set; }

    }
}

File: RetailStation.Angular/src/app/components/Admin/models/SliderModel.ts
import { CreatorModel } from "src/app/components/Shared/models/CreatorModel";

export interface SliderModel extends CreatorModel {
    sliderId: number | null;
    title: string;
    description: string | null;
    image: File | null;
    imageUrl: string; orderNo: number;
    isActive: boolean;
    totalCount?: number;
}